packages/schemas/src/foundations/jsonb-types/one-time-tokens.ts

```diff
@@ -0,0 +1,25 @@
+import { type ToZodObject } from '@logto/connector-kit';
+import { z } from 'zod';
+
+export type OneTimeTokenContext = {
+  // Declare the appIds that users can sign-up to.
+  applicationIds?: string[];
+  // Used for organization JIT provisioning.
+  jitOrganizationIds?: string[];
+};
```

Comment:
Should we add `& Record<string, unknown>` for more flexibility and future scalability?